packages/editor/src/components/ComponentWrapper.tsx

```diff
@@ -156,6 +158,7 @@ export function useComponentWrapper(services: EditorServices): ComponentWrapperT
 
     const onClickWrapper = (e: React.MouseEvent<HTMLElement>) => {
       e.stopPropagation();
+      setRightTabIdx(0);
```

Comment:
Don't do setIndex here. There may be other place select component. Watch `selectedComponentId` change in `editorStore`.